File: recipe-site/app/models/Recipe.ts

export default class Recipe {
    recipeId: number;
    name: string;
    instructions: string;
    prepTimeMinutes: number;
    cookTimeMinutes: number;

    constructor(
        recipeId: number,
        name: string,
        instructions: string,
        prepTimeMinutes: number,
        cookTimeMinutes: number
    ) {
        this.recipeId = recipeId;
        this.name = name;
        this.instructions = instructions;
        this.prepTimeMinutes = prepTimeMinutes;
        this.cookTimeMinutes = cookTimeMinutes;
    }
}

File: java-recipes/src/main/java/com/recipes/models/Recipe.java
package com.recipes.models;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
public class Recipe {
    private Integer recipeId;
    private String name;
    private String instructions;
    private Integer prepTimeMinutes;
    private Integer cookTimeMinutes;
    private Integer totalCalories;
    private String quantity;
    private Integer avatarId;

    @Override
    public String toString() {
        return "Recipe{" +
                "recipeId=" + recipeId +
                ", name='" + name +
                '}';
    }
}
